docs/excel/custom-functions-best-practices.md

```diff
@@ -37,23 +37,6 @@ If you are testing your add-in in Office on Windows, you should enable **[runtim
 
 To report feedback to the Excel Custom Functions team about this method of troubleshooting, send the team feedback. To do this, select **File | Feedback | Send a Frown**. Sending a frown will provide the necessary logs to understand the issue you are hitting.
 
-## Debugging
-
-Currently, the best method for debugging Excel custom functions is to first [sideload](../testing/sideload-office-add-ins-for-testing.md) your add-in within **Excel Online**. You can then debug your custom functions by using the [F12 debugging tool native to your browser](../testing/debug-add-ins-in-office-online.md) in combination with the following techniques:
-
-- Use `console.log` statements within your custom functions code to send output to the console in real time.
-
-- Use `debugger;` statements within your custom functions code to specify breakpoints where execution will pause when the F12 window is open. For example, if the following function runs while the F12 window is open, execution will pause on the `debugger;` statement, enabling you to manually inspect parameter values before the function returns. The `debugger;` statement has no effect in Excel Online when the F12 window is not open. Currently, the `debugger;` statement has no effect in Excel for Windows.
-
-    ```js
-    function add(first, second){
-      debugger;
-      return first + second;
-    }
-    ```
-
-If your add-in fails to register, [verify that SSL certificates are correctly configured](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) for the web server that's hosting your add-in application.
-
 ## Associating function names with JSON metadata
 
 As described in the [custom functions overview](custom-functions-overview.md) article, a custom functions project must include both a JSON metadata file and a script (either JavaScript or TypeScript) file to form a complete function. For a function to work properly, you'll need to bind the name of the function in the script file to the id listed in the JSON file. This process is called association. Make a note to include associations at the end of your JavaScript code files; otherwise, your functions will not work.
```

Comment:
The sentences starting with  "For a function to work properly..." could be more concise. Something like: "For a function to work properly, you need to associate the id with the JavaScript implementation. Make sure there is an association, otherwise the function will not be called."  Also, there is no requirement that the calls to CustomFunctions.associate() are at the end.